static/js/src/advantage/react/components/Subscriptions/Notifications/Notifications.tsx

```diff
@@ -28,6 +30,12 @@ const Notifications = () => {
           subscriptions
         </Notification>
       ) : null}
+      {offers?.length > 0 ? (
+        <Notification data-test="offers" inline severity="information">
+          You have one or more Ubuntu Advantage offers to view.{" "}
+          <a href={urls.advantage.offers}>View your offers.</a>
```

Comment:
Can you use the actions prop for the notification component? https://canonical-web-and-design.github.io/react-components/?path=/story/notification--actions

Reply:
Done!